manifests/backend.pp

```diff
@@ -77,6 +82,8 @@
     $instance_name = "haproxy-${instance}"
     $config_file = inline_template($haproxy::params::config_file_tmpl)
   }
+  include haproxy::globals
+  $_sort_options_alphabetic = pick($sort_options_alphabetic, $haproxy::globals::sort_options_alphabetic)
```

Comment:
It is impossible because you can use the `haproxy` module without using the `haproxy` class, only using  `haproxy::instance` resources.